Sources/HashTreeCollections/HashNode/_HashTreeStatistics.swift

```diff
@@ -115,12 +115,14 @@ extension _HashNode {
 
       let objectHeaderSize = 2 * MemoryLayout<Int>.stride
 
+#if !os(OpenBSD)
       // Note: for simplicity, we assume that there is no padding between
       // the object header and the storage header.
       let start = _getUnsafePointerToStoredProperties(self.raw.storage)
       let capacity = self.raw.storage.capacity
       let end = $0._memory + capacity * MemoryLayout<_RawHashNode>.stride
       stats.grossBytes += objectHeaderSize + (end - start)
+#endif
```

Comment:
Huh; I don't think I actually intended this to invoke `malloc_size`. The byte capacity we initialize above really should suffice on all platforms:

```suggestion
      // Note: for simplicity, we assume that there is no padding between
      // the object header and the storage header.
      let start = _getUnsafePointerToStoredProperties(self.raw.storage)
      let end = $0._memory + $0.byteCapacity
      stats.grossBytes += objectHeaderSize + (end - start)
```

Reply:
Sounds good; tested. I'll commit your suggestion.